src/context/AuthContext.tsx

```diff
@@ -13,9 +13,7 @@ const networkId = (import.meta as any).env.VITE_NETWORK_ID;
 // Assuming you have the necessary imports for the types
 interface DerivedAddressParam {
```

Comment:
Can we use a union type here so a BTC `type` only requires `btcNetworkId`?

Reply:
Good point :) Changed it